native/src/main/java/io/xlibb/pipeline/ErrorUtils.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2025 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package io.xlibb.pipeline;
+
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.values.BError;
+
+import static io.ballerina.runtime.api.utils.StringUtils.fromString;
+
+public class ErrorUtils {
+
+    public static final String ERROR_TYPE = "Error";
```

Comment:
If this is not used anywhere else, lets make it private or at least package-private.

Reply:
Fixed via [6f26c5c](https://github.com/xlibb/module-pipeline/pull/2/commits/6f26c5ce6047dc7a9202745b3f39e7c188f578b4)